src/UXClient/Models/TimeSeriesEvent.ts

```diff
@@ -11,7 +11,7 @@ class TimeSeriesEvent {
             let utcOffsetDate = Utils.offsetUTC(new Date(Date.parse(rawEvent['timestamp ($ts)'].split("Z").join(""))));
             rawEvent[offsetName + "_" + type] = {
                 name: offsetName,
-                value: Utils.timeFormat(true, true, offset, true, null, null, locale)(utcOffsetDate),
+                value: () => Utils.timeFormat(true, true, offset, true, null, null, locale)(utcOffsetDate),
```

Comment:
This function call is wild: true, true, **offset**, true, null, null, locale

Reply:
yeahhh in the future i think we should move to making function calls that have a lot of params like this to take a single object that is strongly typed